components/elm/src/biogeophys/BalanceCheckMod.F90

```diff
@@ -348,17 +353,80 @@ subroutine ColWaterBalanceCheck( bounds, num_do_smb_c, filter_do_smb_c, &
        do fc = 1,num_do_smb_c
           c = filter_do_smb_c(fc)
           g = col_pp%gridcell(c)
+          write(iulog,*)'WARNING:  glc_dyn_runoff_routing = ', glc_dyn_runoff_routing(g)  ! TKT
           if (glc_dyn_runoff_routing(g)) then
              errh2o(c) = errh2o(c) + qflx_glcice_frz(c)*dtime
              errh2o(c) = errh2o(c) - qflx_glcice_melt(c)*dtime
           endif
        end do
-
+       
        found = .false.
        do c = bounds%begc, bounds%endc
+          !! === TKT
+          !g = col_pp%gridcell(c)    !TKT
+          !t = col_pp%topounit(c)    !TKT
+          !l = col_pp%landunit(c)   !TKT
+          !mxElv = grc_pp%MaxElevation(g)
+          !grdElv = grc_pp%elevation(g)
+          !topoElv = top_pp%elevation(t)
+          !numt_pg = grc_pp%ntopounits(g)
+          !!=== end TKT
+          
```

Comment:
Are the commented lines here and below for debugging? Resolve/remove before integration.

Reply:
@thorntonpe I have removed the commented lines in my latest commit.